packages/sdk/src/router/v6/permit2.ts

```diff
@@ -0,0 +1,30 @@
+import { AllowanceTransfer } from "@uniswap/permit2-sdk";
+import * as Sdk from "@reservoir0x/sdk/src";
+
+export const generatePermitSignature = async (
```

Comment:
Let's instead create the permit handling logic in the `permits/permit2.ts` file, containing similar logic to `seaport`: https://github.com/reservoirprotocol/core/blob/main/packages/sdk/src/router/v6/permits/seaport.ts